api/proto/config/appconfig.proto

```diff
@@ -49,9 +50,14 @@ message AppInstanceConfig {
     // App Instance initialization configuration data provided by user
     // This will be used as "user-data" in cloud-init
     // Empty string will indicate that cloud-init is not required
+    // to be deprecated, use cipherData instead
     string userData = 11;
 
     // Config flag if the app-instance should be made accessible
     // through a remote console session established by the device.
     bool remoteConsole = 12;
+
+    // contains the cipher payload
+    // CipherBlock cipherData = 13;
```

Comment:
Shouldn't this be included as opposed to commented out?

Reply:
this is the yetus failure. I still do not have any clue, why it breaks here. For now, I have just commented out to proceed. Will work on it.